### PR TITLE
Improve clarity in the decision making process

### PR DESCRIPTION
**Pull request by benjagm**
Adding a quick introduction to the decision making process to improve clarity

**GitHub Issue:** # / NA

**Summary**: I found the decision making process a bit confusing the first time I tried to follow the voting process. With this change I try to add some more clarity in that section.

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**

No

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -31,6 +31,12 @@ There may be other grounds for removal from the TSC, such as seriously violating
 
 ## Decision Making
 
+The JSON Schema Project uses 2 different mechanism for decision making:
+
+1. **Consensus**: The JSON Schema Project tries by default to find consensus in all the strategic decision by using the consensus mechanism described below. 
+
+2. **Voting**: It is ok use the voting process for those cases when the discussion has already occurred so we can use this simple yet powerful mechanism.
```
Comment (Relequestual) [inline GOVERNANCE.md:38]:
```suggestion
2. **Voting**: It is OK use the voting process for those cases when the discussion has already occurred or discussion and discovery is not required. Voting is a simple yet powerful mechanism. 
```

**Review by Relequestual — COMMENTED** [review]
Suggested a slight change.

**Review by gregsdennis — APPROVED** [review]
I agree with @Relequestual's changes.

**Comment by benjagm** [comment]
Thanks for your reviews and feedback!

**Pull request merged**